scripts/gh-projects/examples/matchline/create-issues.sh

```
@@ -0,0 +1,264 @@
+F=$(prep_body "$SCRIPT_DIR/phase-1/c2.md" "$P1_NUM" "$P1_C1_NUM")
+read P1_C2_URL P1_C2_NUM _ <<<"$(create_child "Experience Unit extraction endpoint (pasted resume → ExperienceUnit[])" \
+  "$F" "matchline,phase-1,agent-action" "$P1_NUM")"
+echo "  C2: $P1_C2_URL"
+
+F=$(prep_body "$SCRIPT_DIR/phase-1/c3.md" "$P1_NUM" "$P1_C2_NUM")
+read P1_C3_URL P1_C3_NUM _ <<<"$(create_child "Unit Review surface (list, filter, inline edit, approve/reject, manual add)" \
+  "$F" "matchline,phase-1,agent-action" "$P1_NUM")"
+echo "  C3: $P1_C3_URL"
+
+F=$(prep_body "$SCRIPT_DIR/phase-1/c4.md" "$P1_NUM" "$P1_C1_NUM")
+read P1_C4_URL P1_C4_NUM _ <<<"$(create_child "Job Requirement parsing endpoint (pasted JD → JobRequirementUnit[])" \
```

Shellcheck (0.11.0)

[info] read without -r will mangle backslashes.

(SC2162)